sdk/search/Azure.Search.Documents/src/Generated/Models/EdgeNGramTokenFilter.Serialization.cs

```diff
@@ -12,79 +12,5 @@ namespace Azure.Search.Documents.Indexes.Models
 {
     public partial class EdgeNGramTokenFilter : IUtf8JsonSerializable
     {
-        void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
-        {
-            writer.WriteStartObject();
-            if (MinGram != null)
-            {
-                writer.WritePropertyName("minGram");
-                writer.WriteNumberValue(MinGram.Value);
-            }
-            if (MaxGram != null)
-            {
-                writer.WritePropertyName("maxGram");
-                writer.WriteNumberValue(MaxGram.Value);
-            }
-            if (Side != null)
-            {
-                writer.WritePropertyName("side");
-                writer.WriteStringValue(Side.Value.ToSerialString());
-            }
-            writer.WritePropertyName("@odata.type");
-            writer.WriteStringValue(ODataType);
-            writer.WritePropertyName("name");
-            writer.WriteStringValue(Name);
-            writer.WriteEndObject();
-        }
-
-        internal static EdgeNGramTokenFilter DeserializeEdgeNGramTokenFilter(JsonElement element)
```

Comment:
This also looks the same from a quick glance?

Reply:
It is, by design.